data/governments.txt

```diff
@@ -193,6 +193,12 @@ government "Deep Security"
 government "Derelict"
 	"fine" 0
 
+government "Derelict (Boardable)"
+	"display name" "Derelict"
```

Comment:
I imagine these are (mostly) derelict merchant ships, in which case we should use swizzle 5.
```suggestion
government "Derelict (Boardable)"
	swizzle 5
	"display name" "Derelict"
```
But these ships are completely abandoned, so maybe they look like they've had a bit more wear and tear.
```suggestion
government "Derelict (Boardable)"
	swizzle 14
	"display name" "Derelict"
```
![image](https://github.com/endless-sky/endless-sky/assets/17688683/29c99ac7-cbba-47b1-8ef9-b59606d8e8da)

Or we could go for something with no color, which gives the ship more of an appearance of having had the life drained out of it. It's also reminiscent of a neutral unit in an RTS game, and works for cases where the derelict ships aren't merchant ships (such as for the derelict pirates fleet).
```suggestion
government "Derelict (Boardable)"
	swizzle 10
	"display name" "Derelict"
```
![image](https://github.com/endless-sky/endless-sky/assets/17688683/aacb5960-22f7-4470-ac63-b8d6a38842a0)

Reply:
Ooh, I like the gray.